celestia/chain.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "celestia",
+  "chain_id": "celestia",
+  "pretty_name": "Celestia",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "celestia",
+  "daemon_name": "celestia-appd",
+  "node_home": "$HOME/.celestia-app",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "utia",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.4,
+        "average_gas_price": 0.1,
+        "high_gas_price": 0.2
+      }
```

Comment:
It appears there might be a typo in the `low_gas_price` value.

Reply:
should be good now?

Reply:
```
"low_gas_price": 0.4,
"average_gas_price": 0.1,
"high_gas_price": 0.2
```

Does not really make sense?